### DISCUSSION TITLE
Sleep and dim on all pages

### DISCUSSION THREAD

**Discussion by denes44** [Blackymas/NSPanel_HA_Blueprint#760] [Ideas]
My next thing I want to work on is enable sleep mode and dim mode on every page.
Right now the display only goes to sleep or dims down the display brightness if it's on the home page. I want to change it, so the display goes to sleep/dims the display even if I leave it on another page.

Before this I need PR #740, #758 and #759 merged, because for this to work, I need to edit all the pages of the nextion file, and it would be pretty hard to merge them.

**Comment by edwardtfn** [comment]
I'm closing this discussion as we have released v4.0 (#1057).
Please add a feature request for the missing things.

And thanks a lot for your support.
